src/Microsoft.AspNet.FileProviders.Combined/CombinedDirectoryContents.cs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.FileProviders
```

Comment:
@Eilon should these go under `Microsoft.AspNet.FileProviders.Combined`? The other two file providers are also placed under this namespace.